### PR TITLE
Updating docs with Kafka scaling steps

### PR DESCRIPTION
**Pull request by sansaid**

### CODE REVIEW

File: docs/first_time_setup.md [hunk 1]
```diff
@@ -7,50 +7,62 @@ Before starting, it's recommended that you run all of these steps [in VS Code an
...
+   ```sh
+   cd apps/kafka
+   flyctl scale memory -a promptu-kafka-eds 2048
+   ```
+5. **Prepare Github Workflow**
+   1. In your forked Github repo, go to _Settings > Security > Secrets > Actions_ and create the following repository secrets:
+      - (**Sensitive**) `FLY_API_TOKEN` (value secured from step `2.4.`)
+   2. Update your `fly.toml` files to include the suffix you provided in step `3.4` (if you chose `paper_mache` as your suffix, then your app name will be `promptu-paper_mache` for the `ui` component and `promptu-api-paper_mache` for the `api` component)
+6. **Raise your first PR and merge it into `main` to build your infrastructure and deploy your apps** - now that all the scaffolding has been set up, it's time to add some bricks! Merging your first PR will deploy your application to fly.io for the first time! All subsequent PRs will not only deploy the latest changes to fly.io, but it will also update your infrastructure through Terraform Cloud. But we're not done yet - we still need to secure access to our DB!
```
Comment (BerniVarga) [inline docs/first_time_setup.md:55]: Should we go with eds/main for now? I'm also thinking that maybe even after the workshop, leaving the repo with two branches (we could make eds/main protected against deletion if it isn't) - would help people orienteer easier? Could feel overwhelming doing both of the setup at the same time for someone who is new to infrastructure.

File: docs/first_time_setup.md [hunk 2]
```diff
@@ -7,50 +7,62 @@ Before starting, it's recommended that you run all of these steps [in VS Code an
...
+7. **Secure your promptu-api to MongoDB Atlas**
+   1. Right now, your DB can be accessed by anyone in the world! We need to restrict this so only our fly.io can communicate with it. Find out the public IP address for the applications `feeder-api` and `db-updater` by using `flyctl ssh issue` to issue an SSH key for your fly.io aap (entering `/home/vscode/.ssh/promptu-feeder-api-fly-io` and `/home/vscode/.ssh/promptu-db-updater-fly-io` respectively as your path to store the keys if you're in a devcontainer - otherwise, you have to supply the absolute path of your `~/.ssh` directory, followed by any prefix name you want e.g. `promptu-api-fly-io`) and then run `flyctl ssh console` from within the `apps/api` directory
+   2. Once inside, run the following commands in order to install `dig` and to find out your app's public IP address - record this IP address
+   ```sh
+   apt update
+   apt install -y dnsutils
+   dig +short txt ch whoami.cloudflare @1.0.0.1
+   ```
```
Comment (BerniVarga) [inline docs/first_time_setup.md:63]: Maybe we can add a: "Note: the ip address shown on the fly io UI is different than the one obtained through the dig command, please use the public ip". (If the one on the UI is the private IP, we can say that too.)

**Review by BerniVarga — DISMISSED** [review]
Added some minor comments, nothing is blocking :)

File: docs/first_time_setup.md [hunk 3]
```diff
@@ -7,50 +7,62 @@ Before starting, it's recommended that you run all of these steps [in VS Code an
...
+   7. In your MongoDB Atlas `promptu` project, go to _Deployment > Database_ and click on _Connect_ on your `promptu-db`. Select _Connect your application_ and copy the connection string up until (but not including!) the URL path and query strings. So if you're connection string looks like `mongodb+srv://promptu:<password>@promptu-db.p4jpncm.mongodb.net/?retryWrites=true&w=majority`, you only need `mongodb+srv://promptu:<password>@promptu-db.p4jpncm.mongodb.net`.
+   8. Replace `<password>` in the connection string from `3.7.` with the value secured from `3.6.` - keep this value secure!
+   9. Use the `flyctl` CLI tool to create a `PROMPTU_MONGODB_URL` secret with your MongoDB connection URL from `3.8.` with the following script - you have to be in the same directory as the each service's `fly.toml` (**note:** make sure you're running this from a safe environment or from within a script, as your secret will be preserved within your shell history if run directly in your shell environment):
+
+   ```sh
+   cd apps/feeder-api
```
[the remark this answers is not in the document]
Comment (sansaid) [reply docs/first_time_setup.md:38]: Yeah, I understand having less to read, but I wanted it to be possible for them to also copy and paste the commands mostly. What are your thoughts on that?

File: docs/first_time_setup.md (hunk 1 above)
Comment (sansaid) [reply docs/first_time_setup.md:55]: The `main` branch is more general to their setup and not specific to ours, but I'll make that more clear

File: docs/first_time_setup.md (hunk 2 above)
Comment (sansaid) [reply docs/first_time_setup.md:63]: Noted!

**Pull request merged**